packages/react-storage/src/components/StorageBrowser/do-not-import-from-here/actions/listLocationsAction.ts

```diff
@@ -34,6 +39,19 @@ const shouldExclude = <T extends Permission>(
     ? exclude === permission
     : exclude.includes(permission);
 
+// FIXME: temporary fix until we use the list action in actions folder
```

Comment:
Sorry i mis context here. Do we mean bringing this in temporarily for now?

Reply:
Yes, this should be replaced by a single implementation when we migrate off this folder https://github.com/aws-amplify/amplify-ui/blob/37a05193693d4948377c3335cd072485f01bd885/packages/react-storage/src/components/StorageBrowser/actions/handlers/utils.ts#L48